.config/ansible-lint.yml

```diff
@@ -0,0 +1,5 @@
+# .config/ansible-lint.yml
+warn_list:  # or 'skip_list' to silence them completely
+  - experimental  # all rules tagged as experimental
+  - latest[git]  # Result of the command may vary on subsequent runs.
```

Comment:
This is idempotence check on git commands, and would fail on the presentation deployment playbook, which deploys on idr0-slot3.
Did not manage to specifically disable that check on this file.